code/game/turfs/turf.dm

```diff
@@ -130,6 +130,11 @@
 	if (!mover)
 		return 1
 
+	var/obj/effect/unwall_field/UF = locate(obj/effect/unwall_field) in src //For traitorous mimes
+	if(UF)
+		UF.to_bump(mover, 1)
```

Comment:
Sorry, didn't see this when we were talking. What's the `1` for?

Reply:
It's in the to_bump s below so probably something